src/DynamoCore/Graph/Notes/NoteModel.cs

```diff
@@ -26,18 +26,25 @@ public string Text
             }
         }
 
-        private NodeModel pinNode;
-        public NodeModel PinNode
+        private NodeModel pinnedNode;
+
+        /// <summary>
+        /// When using the pin to node command, node model to which note is pinned to
```

Comment:
not sure this is very clear, could it maybe be something like `NodeModel which this Note is pinned to` possibly followed by a brief description about what it means that they are pinned?